api/nameguard/nameguard.py

```diff
@@ -440,7 +440,9 @@ def _inspect_confusable(self, grapheme: InspectorConfusableGraphemeResult) -> Co
             is_canonical=False,
         )
 
-    async def secure_primary_name(self, address: str, network_name: str) -> SecurePrimaryNameResult:
+    async def secure_primary_name(
+        self, address: str, network_name: str, return_nameguard_report: bool = True
```

Comment:
Ah interesting, here we default to `True` but in the SDK we default to false? How do you suggest we align this?